components/src/components/upload_model.py

```diff
@@ -25,26 +25,27 @@
 )
 def upload_model(
     model: Input[Model],
-    test_dataset: Input[Dataset],
+    test_data: Input[Dataset],
     model_evaluation: Input[Metrics],
     vertex_model: Output[VertexModel],
-    project_id: str,
-    project_location: str,
+    project: str,
+    location: str,
     model_name: str,
     eval_metric: str,
     eval_lower_is_better: bool,
     pipeline_job_id: str,
     serving_container_image: str,
+    model_description: str = None,
     evaluation_name: str = "Imported evaluation",
```

Comment:
Not important for this story, but I'd suggest some of these variables could also be eliminated